### ISSUE TITLE
QA for desi_boot trace/wavelength solution

### ISSUE THREAD

**Issue by julienguy** [desihub/desispec#228]
We have from time to time bad solutions for the trace/wavelength solution.
It would be useful to have a QA plot that compares the current solution with a reference one and raises a flag if an anomalous difference is found.

**Comment by profxj** [comment]
I think I should prefer a test that compares aspects of the solution (e.g. the dispersion)
against neighboring fibers or the full set of other solutions for that frame.   That identifies
individual fiber failures without relying on a reference.  Or are you worried about a
major break down (many fibers)?

> On Jun 24, 2016, at 5:07 AM, julienguy <notifications@github.com> wrote:
> 
> We have from time to time bad solutions for the trace/wavelength solution.
> It would be useful to have a QA plot that compares the current solution with a reference one and raises a flag if an anomalous difference is found.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/desihub/desispec/issues/228>, or mute the thread <https://github.com/notifications/unsubscribe/AD0BjUM7unPvhldb_U9jT2aJwCyiHZ6jks5qO8h3gaJpZM4I9tY_>.
> 
> !DSPAM:826,576d227a200331669113870!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)

**Comment by julienguy** [comment]
Agreed. For instance delta X (mean wavelength, or mean Y) from neighboring fibers showed an anomaly in one of bootcalib PSF. It was missing a fiber (see bootcalib PR).

**Comment by rkehoe** [comment]
Your preference here seems fine to me.  But in general we will have other 
instances where a reference will be needed for the QA algorithms.  We are 
planning such for algorithms for the test-stand, for instance.  So it will 
be important to acknowledge and preserve that option.

--------------------------------------------------------------------------
Robert Kehoe					      (214)768-1793 (off.)
Professor of Physics
Department of Physics				       (214)768-4095 (fax)
Southern Methodist University			     kehoe@physics.smu.edu
Dallas, TX 75275		          http://www.physics.smu.edu/kehoe
